config.lua

```diff
@@ -169,18 +169,66 @@ Config.AmmoLabels = {
     ["AMMO_SNIPER"] = "Large caliber bullet",
 }
 
+Config.SpeedFines = {
+    [1] = {
+        fine = 50,
+        speedAvg = {1,10}
+    },
+    [2] = {
+        fine = 150,
+        speedAvg = {10,20}
+    },
+    [3] = {
+        fine = 500,
+        speedAvg = {20,300}
+    }
+}
+
```

Comment:
Just a thought on this. This kind of takes away RP for police officers as the player is fined automatically without any police interaction. This could be ok, however, I would suggest flagging the plate for speeding and maybe triggering a 911 to notify the police why the plate was flagged and what speed was caught on camera.

Reply:
Nice suggestion. Will do that ASAP